mouse-look.js

```diff
@@ -43,6 +45,14 @@ WL.registerComponent('mouse-look', {
             }
         }.bind(this));
 
+        if(this.pointerLockOnClick){
+            WL.canvas.addEventListener('mousedown', function() {
+                WL.canvas.requestPointerLock = WL.canvas.requestPointerLock ||WL.canvas.mozRequestPointerLock ||WL.canvas.webkitRequestPointerLock;
+                // Ask the browser to lock the pointer
+                WL.canvas.requestPointerLock();
+            }.bind(this));
```

Comment:
You don't need the `.bind(this)` here, only for members of the component :)

Reply:
Besides, just to explain the arrow function: `() => {}` is the equivalent of `function(){}.bind(this)`, it's basically syntactic sugar.

Reply:
New  info ! @DavidPeicho